### PR TITLE
Adding tests to Util

### LINKED ISSUE #126: Possible Testing Schemes
Viz doesn't have many tests because the products are visually as expected or not, but how does matplotlib or cartopy have a testing scheme?

### PR DESCRIPTION
**Pull request by jukent**
## PR Summary
First pass at tests for util.py
Closes #126 

## PR Checklist
**General**
- [x] Make an issue if one doesn't already exist
- [x] Link the issue this PR resolves by adding `closes #XXX` to the PR description where XXX is the number of the issue.
- [x] Add a brief summary of changes to `docs/release-notes.rst` in a relevant section for the next unreleased release. Possible sections include: Documentation, New Features, Bug Fixes, Internal Changes, Breaking Changes/Deprecated
- [x] Add appropriate labels to this PR
- [x] Make your changes in a forked repository rather than directly in this repo
- [x] Open this PR as a draft if it is not ready for review
- [x] Convert this PR from a draft to a full PR before requesting reviewers
- [x] Passes `precommit`. To set up on your local, run `pre-commit install` from the top level of the repository. To manually run pre-commits, use `pre-commit run --all-files` and re-add any changed files before committing again and pushing.

### CODE REVIEW

**Comment by jukent** [comment]
Currently there are some place holder functions in here and I did not generate the requisite images yet, just creating this PR so that my work so far is visible.

**Comment by jukent** [comment]
I would love to replace some of the datafiles accessed will simpler dummy data, but because of the lat/lon grids and skewt plots, it seemed easier to access these small datafiles.

File: docs/release-notes.rst [hunk 1]
```diff
@@ -15,6 +15,9 @@ Internal Changes
 Testing
 ^^^^^^^
 * Add basic testing infrastructure by `Katelyn FitzGerald`_ and `Julia Kent`_ in (:pr:`233`)
+* Test `viz.taylor` by `Julia Kent`_ in (:pr:`234`)
+* Test `viz.util` by `Julia Kent`_ in (:pr:`239`)
```
Comment (kafitzgerald) [inline docs/release-notes.rst:19]:
```suggestion
* Add tests for `viz.taylor` by `Julia Kent`_ in (:pr:`234`)
* Add tests for `viz.util` by `Julia Kent`_ in (:pr:`239`)
```

File: tests/test_util.py [hunk 2]
```diff
@@ -1,13 +1,314 @@
...
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import xarray as xr
+import pandas as pd
+from metpy.units import units
+import metpy.calc as mpcalc
 
-from geocat.viz.util import truncate_colormap
+import geocat.viz.util as gv
```
Comment (kafitzgerald) [inline tests/test_util.py:12]: Was there a reason for switching this up?  The convention I've typically seen for code being tested is the explicit "from x import y" for all tested functions and classes.  Generally at the bottom of the imports section.

**Review by kafitzgerald — CHANGES_REQUESTED** [review]
Sorry for the individual suggestions before - I meant to include those in the review.  

This looks like a great start to me!  Obviously, things at could be improved / added, but I think just getting this merged ASAP probably makes sense.

I think my main question/concern is about the imports (see the inline comment).  

And then a minor thing - should the tests/test.ipynb be removed?  It looks like an empty file that got added by accident.

File: tests/test_util.py (hunk 2 above)
Comment (jukent) [reply tests/test_util.py:12]: I did not realize that, I thought importing every single util function would take up a lot of space, so I imported the module instead. I see how that would make it more clear right away.

**Review by kafitzgerald — APPROVED** [review]
🥳

**Pull request merged**